packages/react-geo/src/styles.css

```diff
@@ -1 +1,4 @@
+@import '@maplibre/maplibre-gl-geocoder/dist/maplibre-gl-geocoder.css';
+@import 'maplibre-gl/dist/maplibre-gl.css';
+@import 'maplibre-gl-js-amplify/dist/public/amplify-geocoder.css';
```

Comment:
We should write some unit tests to prevent regressions and make sure these styles are applied as expected. Re-ordering is nice but now they're not in the order they were in the react pkg, have we thoroughly verified it works as before?

Reply:
+1

Reply:
Yes. We have a meeting today to discuss how to proceed with this. Will update PR based on that.

Reply:
I have verified that the new order of css imports works the same.